### PR TITLE
LPS-138193

### PR DESCRIPTION
**Pull request by daiazius**

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
Apparently `LocalFile.Login#TokenBasedSSOSmoke` is not related to this PR because of [LRQA-83146](https://liferay.atlassian.net/browse/LRQA-83146).

**Comment by daiazius** [comment]
ci:reevaluate:1377401_14844

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/persistence/listener/OpenIdConnectConfigurationModelListener.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			properties.get("tokenRefreshOffset"));
+
+		if (tokenRefreshOffset < 30) {
+			throw new ConfigurationModelListenerException(
+				"Token refresh offset needs to be at least 30 seconds",
+				OpenIdConnectConfiguration.class,
+				OpenIdConnectConfigurationModelListener.class, properties);
+		}
+
+		properties.put("tokenRefreshOffset", tokenRefreshOffset);
```
Comment (stian-sigvartsen) [inline modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/persistence/listener/OpenIdConnectConfigurationModelListener.java:41]: Hi @daiazius. Is this line needed? The value is unchanged.

**Comment by stian-sigvartsen** [comment]
Just commented on one small thing. Otherwise this PR is good to go. Thanks @daiazius !

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/persistence/listener/OpenIdConnectConfigurationModelListener.java (hunk 1 above)
Comment (daiazius) [reply modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/persistence/listener/OpenIdConnectConfigurationModelListener.java]: Yeah, it is unnecessary, I'll remove it

**Comment by daiazius** [comment]
ci:forward

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:forward:force

**Pull request closed without merging**